syft/pkg/erlang.go

```diff
@@ -7,3 +7,8 @@ type ErlangRebarLockEntry struct {
 	PkgHash    string `mapstructure:"pkgHash" json:"pkgHash"`
 	PkgHashExt string `mapstructure:"pkgHashExt" json:"pkgHashExt"`
 }
+
+type ErlangOTPApplication struct {
```

Comment:
the metadata type isn't really needed if it is exclusively for name and version (which is already on package)